hips/draw/simple.py

```diff
@@ -1,5 +1,156 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HiPS tile drawing -- simple method."""
+import os
+from pathlib import Path
+
+import healpy as hp
```

Comment:
Hmmm ... we said that we would try to `import healpy` only from `hips.utils.healpix`.
I think here you're only accessing it from a method that would better fit on `TileMeta`, or maybe it can go to `hips.utils.healpix` as an extra wrapper function, or even replacing the existing function that computes corners?